.github/workflows/ci.yaml

```diff
@@ -133,6 +133,37 @@ jobs:
         run: sbt ";sjsJUnitTests/test ;sjsCompilerTests/test"
         shell: cmd
 
+  build_docs:
```

Comment:
I was under impression that we want to have a dedicated fast running CI for scala3docs and having it as a step in main build will not be possible.

Reply:
I thought about it and I think something like what we have here is the cleanest way. `build_docs` actually runs concurrently with other "jobs", so it finishes very quickly (in 4 minutes). We might still make it a separate workflow so that we can disable the primary workflow when only Scala3doc has changed.